.github/workflows/roadie.yml

```diff
@@ -31,7 +31,9 @@ jobs:
         run: yarn run build
 
       - name: Set image version
-        run: echo "IMAGE_VERSION=$(cat IMAGE_VERSION)" >> $GITHUB_ENV
+        run: |
+          echo "IMAGE_VERSION=$(git describe)" >> $GITHUB_ENV
+          echo ${{ env.IMAGE_VERSION }} > IMAGE_VERSION
```

Comment:
What does this line do?

Reply:
Writes the image version back to the filesystem so even if the image is mis-labeled later we can always know what is inside

Reply:
Cool!